rakefile

```diff
@@ -267,30 +267,50 @@ rototiller_task :acceptance do |t|
 end
 
 desc 'Run Scale setup and test for gatling'
-rototiller_task :autoscale do |t|
+rototiller_task :autoscale do
   ENV['ENVIRONMENT_TYPE'] = 'gatling'
   ENV['BEAKER_TESTS'] = ENV['BEAKER_TESTS'] || 'tests/Scale.rb'
   ENV['BEAKER_PRESERVE_HOSTS'] = 'always'
   ENV['PUPPET_SCALE_CLASS'] = 'role::by_size::small'
+  ENV['SCALE_TUNE'] = ENV['SCALE_TUNE'] || 'true'
   Rake::Task["performance"].execute
+  Rake::Task["autoscale_copy_log"].execute unless ENV['BEAKER_TESTS'] == ''
 end
 
 desc 'Run Scale setup for gatling'
-rototiller_task :autoscale_setup do |t|
+rototiller_task :autoscale_setup do
   ENV['BEAKER_TESTS'] = ''
   Rake::Task["autoscale"].execute
 end
 
+desc 'Run puppet infrastructure tune setup for gatling'
```

Comment:
looks like it also runs performance at the end.  Should maybe whatever calls this be the thing that calls performance if it wants to continue on?  I dunno.  The overall flow of this rake file is very obfuscated.

Reply:
I've updated the description to: 
```
desc 'Run puppet infrastructure tune setup for gatling on a previously provisioned master
```
This task is only for running standalone to tune (or re-tune) an already provisioned master.

I've updated the NEW_README with more info about the tasks. In a nutshell:
* `autoscale` is the primary task. When run by itself it runs the pre-suite and test.
* `autoscale_setup` provisions and runs the pre-suite by setting `ENV['BEAKER_TESTS'] = ''` and calling the `autoscale` task.
* `autoscale_tune` runs the tune test from the pre-suite against previously provisioned nodes.
* `autoscale_provisioned` runs the standard scale test against previously provisioned nodes.
* `autoscale_provisioned_x` runs variations on the scale test against previously provisioned nodes.
* `autoscale_copy_log` copies the latest log files to the latest perf scale results folder after the beaker run completes.

So you can either run `autoscale` to provision and run the test or `autoscale_setup` to provision and `autoscale_provisioned` or a variation to run the test.